<?xml version="1.0" ?>
<launch>
    <!-- testing if segmentation/record_flag work -->
    <arg name="topic"  	    	default="/record_manager/record_flag /record_manager/segment_num"/>
	<arg name="path_save"           default="/home/asl_team/teleop_recordings"/>	
	<arg name="file_name"           default=""/>

	<node pkg="record_teleop" type="record_manager.py" name="record_manager" output="screen">
		<!-- Pass arguments as parameters to the node -->
		<param name="topic" value="$(arg topic)"/>
		<param name="path_save" value="$(arg path_save)"/>
		<param name="file_name" value="$(arg file_name)"/>
	</node>



</launch>